os/board/imxrt1050-evk/include/board.h

```diff
@@ -113,6 +113,10 @@
 #define IMXRT_PERCLK_CLK_SEL      CCM_CSCMR1_PERCLK_CLK_SEL_IPG_CLK_ROOT
 #define IMXRT_PERCLK_PODF_DIVIDER 9
 #define IMXRT_SEMC_PODF_DIVIDER   8
+#define IMXRT_LPSPI_CLK_SELECT    CCM_CBCMR_LPSPI_CLK_SEL_PLL3_PFD0
+#define IMXRT_LSPI_PODF_DIVIDER   8
+#define IMXRT_USDHC1_CLK_SELECT    CCM_CSCMR1_USDHC1_CLK_SEL_PLL2_PFD0
```

Comment:
Please adjust the spacing to aligned with other definitions.

Reply:
I fixed it
Please check it